scripts/upgrade/upgrade_a_running_module.sh

```diff
@@ -27,5 +27,5 @@ lavad tx gov submit-proposal software-upgrade $UPRADE_NAME --title upgrade --des
 lavad tx gov deposit 1 10000000ulava --from alice --yes --gas-adjustment "1.5" --gas "auto" --gas-prices $GASPRICE
 lavad tx gov vote 1 yes --from alice --yes --gas-adjustment "1.5" --gas "auto" --gas-prices $GASPRICE
 echo "chosen block for upgrade: $BLOCK_HEIGHT_CHOSEN"
-lavad q upgrade plan
+lavad q upgrade plans
```

Comment:
why?

Reply:
because the module's name is "plans"

Reply:
Actually when I looked closely, the "plan" there is not related to my plans module, so the change is wrong. I've undid it